[2024-06-27T15:41:28.000-0400] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T15:41:28.024-0400] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:07:30+00:00 [queued]>
[2024-06-27T15:41:28.035-0400] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:07:30+00:00 [queued]>
[2024-06-27T15:41:28.035-0400] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-27T15:41:28.093-0400] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_kafka_producer> on 2024-06-27 00:07:30+00:00
[2024-06-27T15:41:28.107-0400] {standard_task_runner.py:63} INFO - Started process 186315 to run task
[2024-06-27T15:41:28.111-0400] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'api_cuevana3_movie', 'run_kafka_producer', 'scheduled__2024-06-27T00:07:30+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/movie_recommend.py', '--cfg-path', '/tmp/tmp1l_hv64a']
[2024-06-27T15:41:28.116-0400] {standard_task_runner.py:91} INFO - Job 127: Subtask run_kafka_producer
[2024-06-27T15:41:28.236-0400] {task_command.py:426} INFO - Running <TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:07:30+00:00 [running]> on host walter-VirtualBox
[2024-06-27T15:41:28.482-0400] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='api_cuevana3_movie' AIRFLOW_CTX_TASK_ID='run_kafka_producer' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:07:30+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:07:30+00:00'
[2024-06-27T15:41:28.483-0400] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T15:41:28.504-0400] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-27T15:41:28.504-0400] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/walter/Documentos/rakuntentv/venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/walter/Documentos/rakuntentv/venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/walter/Documentos/rakuntentv/venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/walter/Documentos/rakuntentv/venv/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/walter/Documentos/rakuntentv/venv/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/walter/Documentos/rakuntentv/airflow/dags/movie_recommend.py", line 41, in json_serialization
    producer = Producer(producer_config)
               ^^^^^^^^^^^^^^^^^^^^^^^^^
cimpl.KafkaException: KafkaError{code=_INVALID_ARG,val=-186,str="No such configuration property: "bootstrap_server""}
[2024-06-27T15:41:28.514-0400] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=api_cuevana3_movie, task_id=run_kafka_producer, run_id=scheduled__2024-06-27T00:07:30+00:00, execution_date=20240627T000730, start_date=20240627T194128, end_date=20240627T194128
[2024-06-27T15:41:28.549-0400] {standard_task_runner.py:110} ERROR - Failed to execute job 127 for task run_kafka_producer (KafkaError{code=_INVALID_ARG,val=-186,str="No such configuration property: "bootstrap_server""}; 186315)
[2024-06-27T15:41:28.617-0400] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-27T15:41:28.646-0400] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-27T15:41:28.647-0400] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-27T16:02:33.620-0400] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T16:02:33.657-0400] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:07:30+00:00 [queued]>
[2024-06-27T16:02:33.661-0400] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:07:30+00:00 [queued]>
[2024-06-27T16:02:33.662-0400] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-27T16:02:33.677-0400] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_kafka_producer> on 2024-06-27 00:07:30+00:00
[2024-06-27T16:02:33.686-0400] {standard_task_runner.py:63} INFO - Started process 188967 to run task
[2024-06-27T16:02:33.690-0400] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'api_cuevana3_movie', 'run_kafka_producer', 'scheduled__2024-06-27T00:07:30+00:00', '--job-id', '178', '--raw', '--subdir', 'DAGS_FOLDER/movie_recommend.py', '--cfg-path', '/tmp/tmpg3ti1l9j']
[2024-06-27T16:02:33.691-0400] {standard_task_runner.py:91} INFO - Job 178: Subtask run_kafka_producer
[2024-06-27T16:02:33.798-0400] {task_command.py:426} INFO - Running <TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:07:30+00:00 [running]> on host walter-VirtualBox
[2024-06-27T16:02:33.936-0400] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='api_cuevana3_movie' AIRFLOW_CTX_TASK_ID='run_kafka_producer' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:07:30+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:07:30+00:00'
[2024-06-27T16:02:33.937-0400] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T16:02:34.017-0400] {python.py:237} INFO - Done. Returned value was: None
[2024-06-27T16:02:34.018-0400] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-27T16:02:34.025-0400] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=api_cuevana3_movie, task_id=run_kafka_producer, run_id=scheduled__2024-06-27T00:07:30+00:00, execution_date=20240627T000730, start_date=20240627T200233, end_date=20240627T200234
[2024-06-27T16:02:34.090-0400] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-27T16:02:34.113-0400] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-27T16:02:34.118-0400] {local_task_job_runner.py:222} INFO - ::endgroup::
